content/.template.config/template.json

```diff
@@ -212,7 +212,7 @@
         }
       ],
       "args": {
-        "+x": ["*/*.sh", "*/*/*.sh"]
+        "+x": ["../*/*.sh", "../*/*/*.sh"]
```

Comment:
I'd use `find -exec` instead of trying to guess how many separators could be there.

By the way, I wonder how does templating engine know to first expand this and then pass it to `chmod`? :thinking: